fetch.go

```diff
@@ -68,11 +68,14 @@ func (conn *Conn) fetchResults() ([]*Result, error) {
 		}
 
 		for i := 0; ; i++ {
-			rowCode := C.dbnextrow(conn.dbproc)
-			if rowCode == C.NO_MORE_ROWS {
+			switch rowCode := C.dbnextrow(conn.dbproc); rowCode {
+			case C.NO_MORE_ROWS:
 				break
```

Comment:
This break terminates execution of the switch statement. 
That changes the behavior.
C.NO_MORE_ROWS status should terminate outer for statement.